syne_tune/optimizer/schedulers/searchers/searcher_factory.py

```diff
@@ -12,20 +12,32 @@
 # permissions and limitations under the License.
 import logging
 
-from syne_tune.optimizer.schedulers.searchers.gp_fifo_searcher import GPFIFOSearcher
-from syne_tune.optimizer.schedulers.searchers.constrained_gp_fifo_searcher import (
-    ConstrainedGPFIFOSearcher,
-)
-from syne_tune.optimizer.schedulers.searchers.cost_aware_gp_fifo_searcher import (
-    CostAwareGPFIFOSearcher,
-)
-from syne_tune.optimizer.schedulers.searchers.gp_multifidelity_searcher import (
-    GPMultiFidelitySearcher,
-)
-from syne_tune.optimizer.schedulers.searchers.cost_aware_gp_multifidelity_searcher import (
-    CostAwareGPMultiFidelitySearcher,
-)
 from syne_tune.optimizer.schedulers.searchers.searcher import RandomSearcher
+try:
+    from syne_tune.optimizer.schedulers.searchers.gp_fifo_searcher import (
+        GPFIFOSearcher,
+    )
+    from syne_tune.optimizer.schedulers.searchers.constrained_gp_fifo_searcher import (
+        ConstrainedGPFIFOSearcher,
+    )
+    from syne_tune.optimizer.schedulers.searchers.cost_aware_gp_fifo_searcher import (
+        CostAwareGPFIFOSearcher,
+    )
+    from syne_tune.optimizer.schedulers.searchers.gp_multifidelity_searcher import (
+        GPMultiFidelitySearcher,
+    )
+    from syne_tune.optimizer.schedulers.searchers.cost_aware_gp_multifidelity_searcher import (
+        CostAwareGPMultiFidelitySearcher,
+    )
+except ImportError:
+    print(
```

Comment:
This comment is copy-pasted at three places, it probably makes sense to define it only once as an util.